proposed/extended-coding-style-guide.md

```diff
@@ -158,14 +158,14 @@ declare(strict_types=1);
 
 namespace Vendor\Package;
 
-use Vendor\Package\{ClassA as A, ClassB, ClassC as C};
-use Vendor\Package\SomeNamespace\ClassD as D;
-use Vendor\Package\AnotherNamespace\ClassE as E;
+use Vendor\Package\{ClassA as A, ClassB, ClassC as C};
```

Comment:
What space change is acting up here?

Reply:
I didn't know at first either, but when checking the change it was because the previous spaces were `&NonBreakingSpace;` and my editor automatical changed them to standard space